scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -569,11 +569,14 @@ def handle(self):
                 try:
                     (stdout, stderr) = proc.communicate(timeout=60)
                     # ignore unencodable unicode chars
-                    logger.debug(stdout.decode(errors = "ignore"))
+                    decoded_stdout = stdout.decode(errors="ignore")
+                    decoded_stderr = stderr.decode(errors="ignore")
+                    logger.debug("Flash command stdout: %s", decoded_stdout)
+                    logger.error("Flash command stderr: %s", decoded_stderr)
 
```

Comment:
Do we really want to log error here? It is logged also when there is no error and stderr is empty.

Reply:
How about logging the err to logger.debug? If we're debugging it would be nice to see that it is in fact empty.

Reply:
> `logger.debug` is better. Or you can just check if stderr is not empty, and then log it and log to file (what is couple lines below) .. 

+1

> BTW. maybe we should log to file all the output from flashing, also stdin?

then it should be two separate files, otherwise it will be confusing to see both stdout and stderr not in order of actual output.

Reply:
My goal here was to output the error to stdout since i couldn't easily find the error from the flashing process. This is what helped me find the problem. Absolutely nothing was logged about my previous error to stdout.